test/js/property-dictionary.js

```diff
@@ -65,4 +65,15 @@ suite('PropertyDictionary', function() {
 
     assert.throw(function () {cssPropertyDictionary.getListValueSeparator('height')}, TypeError);
   });
+
+  test('The getValidStyleValuesArray should return an array of constructors for all StyleValue types accepted by that propertty', function() {
```

Comment:
Can leave off "The" at the start.
s/propertty/property (extra t)

'getValidStyleValuesArray should return an array of constructors for all StyleValue types accepted by that property'